README.md

```diff
@@ -109,7 +109,7 @@ After the initial backup, use the following commands:
  - The `ghe-backup` command creates incremental snapshots of repository data,
    along with full snapshots of all other pertinent data stores.
  - The `ghe-restore` command restores snapshots to the same or separate GitHub
-   Enterprise appliance.
+   Enterprise appliance. You must add the backup host's SSH key to the target GitHub Enterprise appliance before using this command.
```

Comment:
@dbussink Not sure if adding a line break is what you were asking me to do, if not let me know and I'll try to correct it.